pkg/plugin/processor/builtin/setField_test.go

```diff
@@ -0,0 +1,132 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package builtin
+
+import (
+	"context"
+	"testing"
+
+	"github.com/conduitio/conduit-commons/opencdc"
+	sdk "github.com/conduitio/conduit-processor-sdk"
+	"github.com/conduitio/conduit/pkg/foundation/cerrors"
+	"github.com/matryer/is"
+)
+
+func TestSetField_Process(t *testing.T) {
+	proc := setField{}
+	var err error
+	ctx := context.Background()
+	testCases := []struct {
+		field  string
+		value  string
+		record opencdc.Record
+		want   opencdc.Record
```

Comment:
If `want` is a `sdk.SingleRecord`, then you can simplify the check to be only `is.Equal(want, got)`.